rllib/core/models/torch/primitives.py

```diff
@@ -295,21 +331,31 @@ def __init__(
             # Then do the Conv2DTranspose operation
             # (now that we have padded and strided manually, w/o any more padding using
             # stride=1).
-            layers.append(
-                nn.ConvTranspose2d(
-                    in_depth,
-                    out_depth,
-                    kernel,
-                    # Force-set stride to 1 as we already took care of it.
-                    1,
-                    # Disable torch auto-padding (torch interprets the padding setting
-                    # as: dilation (==1.0) * [`kernel` - 1] - [`padding`]).
-                    padding=(k_w - 1, k_h - 1),
-                    # Last layer always uses bias (b/c has no LayerNorm, regardless of
-                    # config).
-                    bias=cnn_transpose_use_bias or is_final_layer,
-                ),
+
+            layer = nn.ConvTranspose2d(
+                in_depth,
+                out_depth,
+                kernel,
+                # Force-set stride to 1 as we already took care of it.
+                1,
+                # Disable torch auto-padding (torch interprets the padding setting
+                # as: dilation (==1.0) * [`kernel` - 1] - [`padding`]).
+                padding=(k_w - 1, k_h - 1),
+                # Last layer always uses bias (b/c has no LayerNorm, regardless of
+                # config).
+                bias=cnn_transpose_use_bias or is_final_layer,
             )
+
+            # Initialize, if necessary.
+            if cnn_transpose_initializer:
+                if cnn_transpose_initializer_config:
```

Comment:
Simplify to:
```
if cnn_transpose_initializer_config is not None:
    cnn_transpose_initializer(
        layer.weight, **(cnn_transpose_initializer_config or {})
    )
```

Gets rid of one entire if-else-block.

Reply:
Great catch! Thanks @sven1977 !

Reply:
I think we do not need the `or {}` there